### PR TITLE
Update the release notes descriptions

### PR DESCRIPTION
**Pull request by alina-derkach-oaza**
modified:   docs/release_notes.rst

- [ ] The contributed code is licensed under GPL v2.0
- [ ] Contributor Licence Agreement (CLA) is signed
- [ ] util/update-modules has been ran
- [ ] Documention updated
- [ ] Test suite update

### CODE REVIEW

**Review by patrickbirch — APPROVED** [review]
Good

File: docs/release_notes.rst [hunk 1]
```diff
@@ -33,11 +33,11 @@ Bugs Fixed
 * :jirabug:`PT-1700`: pt-table-checksum REPLICATION_STOPPED exit status not documented
 * :jirabug:`PT-1927`: pt-k8s-debug-collector requires mysql client
 * :jirabug:`PT-1930`: pt-k8s-debug-collector should not attempt collecting PXC information when running against PSMDB
-* :jirabug:`PT-2084`: Changes from PT-80 broke previous pt-stalk functionality (Thanks to Agustín Gallego for reporting this issue)
-* :jirabug:`PT-2102`: pt-mysql-summary should support !include* in config files (Thanks to Yoann La Cancellera for reporting this issue)
-* :jirabug:`PT-2110`: Incorrect PTDEBUG output with --ignore-engines option (Thanks to Kazuya Yokogawa for reporting this issue)
+* :jirabug:`PT-2084`: Changes from PT-80 broke previous pt-stalk functionality (Thanks to Agustín Gallego for reporting and fixing this issue)
+* :jirabug:`PT-2102`: pt-mysql-summary should support !include* in config files (Thanks to Yoann La Cancellera for reporting and fixing this issue)
```
Comment (svetasmirnova) [inline docs/release_notes.rst]:
```suggestion
* :jirabug:`PT-2102`: pt-mysql-summary should support !include* in config files (Thanks to Yoann La Cancellera for fixing this issue)
```

File: docs/release_notes.rst [hunk 2]
```diff
@@ -33,11 +33,11 @@ Bugs Fixed
...
+* :jirabug:`PT-2110`: Incorrect PTDEBUG output with --ignore-engines option (Thanks to Kazuya Yokogawa for reporting and fixing this issue)
 * :jirabug:`PT-2114`: Incorrect casting of BIT columns by pt-archiver
-* :jirabug:`PT-2116`: pt-osc --null-to-not-null description is wrong (Thanks to Yoann La Cancellera for reporting this issue)
+* :jirabug:`PT-2116`: pt-osc --null-to-not-null description is wrong (Thanks to Yoann La Cancellera for reporting and fixing this issue)
```
Comment (svetasmirnova) [inline docs/release_notes.rst]:
```suggestion
* :jirabug:`PT-2116`: pt-osc --null-to-not-null description is wrong (Thanks to Yoann La Cancellera for fixing this issue)
```

**Review by svetasmirnova — CHANGES_REQUESTED** [review]
Please review suggestions.

File: docs/release_notes.rst (hunk 1 above)
Comment (alina-derkach-oaza) [reply docs/release_notes.rst]: updated

File: docs/release_notes.rst (hunk 2 above)
Comment (alina-derkach-oaza) [reply docs/release_notes.rst]: updated

**Review by svetasmirnova — APPROVED** [review]

**Pull request merged**